contracts/erc20/Auction.sol

```diff
@@ -61,42 +83,50 @@ contract Auction is ReentrancyGuardUpgradeable, Composable, IAuctionInitializabl
     uint64 start,
     uint32 length
   ) external override nonReentrant returns (uint256 id) {
```

Comment:
This function does not actually return anything?
Also it creates multiple batches with many IDs - should this `returns (uint256 id)` just be removed?